store/tikv/client.go

```diff
@@ -501,7 +490,8 @@ func (c *rpcClient) createConnArray(addr string) (*connArray, error) {
 	array, ok := c.conns[addr]
 	if !ok {
 		var err error
-		array, err = newConnArray(MaxConnectionCount, addr, c.security)
+		connCount := config.GetGlobalConfig().TiKVClient.GrpcConnectionCount
```

Comment:
`GrpcConnectionCount` could be 0?

Reply:
please check 0 here or in the global config initialization.

Reply:
Fixed.